application/controllers/SurveyAdministrationController.php

```diff
@@ -2309,11 +2309,13 @@ public function actionExpireMultipleSurveys()
         $aResults = array();
         $expires = App()->request->getPost('expires');
         $formatdata = getDateFormatData(Yii::app()->session['dateformat']);
+        Yii::import('application.libraries.Date_Time_Converter', true);
         if (trim($expires) == "") {
             $expires = null;
         } else {
-            $datetimeobj = DateTime::createFromFormat($formatdata['phpdate'] . ' H:i', $expires);
-            $expires = $datetimeobj->format("Y-m-d H:i:s");
+            //new Date_Time_Converter($expires, $formatdata['phpdate'].' H:i');
+            $datetimeobj = new date_time_converter($expires, $formatdata['phpdate'] . ' H:i');
```

Comment:
Class spelling is different, but maybe it doesn't matter? date_time_converter vs Date_Time_Converter.

Reply:
https://stackoverflow.com/questions/5260168/capital-letters-in-class-name-php

Reply:
Not case sensitive. But still.

Reply:
Let us make things straight, just in case

Reply:
Autoloading on linux can broke (but no autoload here)